benchmarks/build.gradle

```diff
@@ -80,3 +79,12 @@ jmh {
         }
     }
 }
+
+// Workaround a bug where the conflict of output path between 'benchmarks.jmh' and 'benchmarks.test' in IDEA
```

Comment:
I haven't seen this before, what is it working around?

Reply:
I think, maybe, this is caused by plugin scala.
If I remove this block I will get the following message from the IntelliJ.
<img width="395" alt="image" src="https://user-images.githubusercontent.com/1866157/85517261-0e26be80-b63a-11ea-87d4-a754fee2066c.png">